app/src/main/java/server/GreetServer.java

```diff
@@ -13,26 +14,19 @@
     private BufferedReader in;
 
     int portNumber;
-    ArrayList<String> routeList = new ArrayList<>();
+    HashMap routeList;
```

Comment:
Best practice to assign the type signature of a `HashMap<k, v>` -
```suggestion
    HashMap<String, Route> routeList;
```
I'm curious, did IntelliJ yell at you about that?

Reply:
Vaguely remember fixing these in the next version/upcoming pull request. Next pull I will make sure all my files have green checks.